src/api/logger-api.ts

```diff
@@ -247,9 +239,9 @@ export class LoggerApi {
    *     {'product': ['product1', 'product2']}
    * Each call to this API should have the first argument
    * as a valid string from the enum SubscriptionState.
-   * @param {SubscriptionState} state
-   * @param {?JsonObject} jsonProducts
    */
-  sendSubscriptionState(state, jsonProducts) {}
+  sendSubscriptionState(
+    state: SubscriptionState,
+    jsonProducts: {[key: string]: string[]}
```

Comment:
Can this be typed more precisely with {product: string[]}?

Reply:
Updated!